FAQ.md

```diff
@@ -199,9 +199,10 @@ Firecracker process remains unaware of the guest shutdown so it lives on.
 Running the `reboot` command in a Linux guest will gracefully bring down the guest
 system and also bring a graceful end to the Firecracker process.
 
-Issuing a `SendCtrlAltDel` action command through the Firecracker API will generate a
-`Ctrl + Alt + Del` keyboard event in the guest resulting in a clean reboot on most
-guest Linux systems.
+On `x86_64` systems, issuing a `SendCtrlAltDel` action command through the
```

Comment:
nit: To be more precise, should we say `hosts` instead of `systems`?

Reply:
I think it's the same since the guest has to run on the same arch as the host. As they are the same, I think it makes sense to use "systems" as an all-encompassing description of the setup.